### PR TITLE
Download PDFs from Wiley service

### PR DESCRIPTION
**Pull request by ehanson8**
#### What does this PR do?
* Add function to download PDF from Wiley author manuscript service
* Add corresponding unit test and fixtures
* Update .gitignore file

#### Helpful background context
This branch will be rebased after #9 and #10 are approved and merged

#### How can a reviewer manually see the effects of these changes?
Run unit tests with `pipenv run pytest`

#### What are the relevant tickets?
- https://mitlibraries.atlassian.net/browse/DLSPP-30

#### Requires Database Migrations?
NO

#### Includes new or updated dependencies?
NO

### CODE REVIEW

File: awd/crossref.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
-import pandas
+import json
+
 import requests
+import smart_open
 
 
 def get_dois_from_spreadsheet(file):
```
Comment (osmandin) [inline awd/crossref.py:7]: Is this method introduced already in the other PR?

**Comment by ehanson8** [comment]
@osmandin This is now ready for review

File: awd/wiley.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import requests
+
+headers = {
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 "
+    "(KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36"
+}
+
+
+def get_wiley_pdf(url, doi):
+    """Get PDF from Wiley server based on a DOI."""
```
Comment (osmandin) [inline awd/wiley.py:10]: Adding a logging statement here will be helpful in debugging.

Comment (ehanson8) [reply awd/wiley.py:10]: I'm handling all logging in the integration function that you'll see in the next PR, but yes, this is one of the potential failure points that I'm tracking

Comment (ehanson8) [reply awd/wiley.py:10]: OK, I'll add that

**Pull request merged**